src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_ConditionalExpressions.vb

```diff
@@ -235,6 +235,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
             End If
 
+            ' Optimize If(left, right) to left.GetValueOrDefault() when left is T? and right is the default value of T
+            If rewrittenLeft.Type.IsNullableType() AndAlso 
+               rewrittenRight.IsDefaultValue() AndAlso
```

Comment:
>IsDefaultValue [](start = 30, length = 14)

We don't need to unwrap identity conversions here, as in C#?

Reply:
If you can identify such a scenario (where the optimization won't kick in because we're not unwrapping identity conversions), that would be a great one to add.